### PR TITLE
some fixes to the vector exersize

### PR DESCRIPTION
**Pull request by pvgenuchten**
removed multilingual collection title
see also https://github.com/geopython/pygeoapi/issues/956

use the gpkg from workshop folder instead of downloading

question: is it a good approach to keep the data file zipped and ask to unzip, alernative is to store unzipped (6mb vs 2mb), also i can lower the number of features to be smaller (it is already a subset)

### CODE REVIEW

File: workshop/content/docs/publish/vector.md [hunk 1]
```diff
@@ -36,43 +32,84 @@ You are going to add a file `firenze-terrains.gpkg` to pygeoapi which is availab
...
-Save the file and restart the service. Navigate to localhost:5000/pygeoapi/collections to evaluate if the new dataset is available.
+Save the file and restart the docker compose. Navigate to localhost:5000/collections to evaluate if the new dataset is available.
 
 !!! tip 
 
     The sqlite driver incidentally has challenges to open the geopackage extension on MacOS. Read [this thread](https://docs.pygeoapi.io/en/latest/development.html#working-with-spatialite-on-osx) or try with an alternative data format.
  
+## pygeoapi as a WFS proxy
+
+An interesting use case for pygeoapi is to provide a OGC API Features interface over existing WFS endpoints. In this scenario you increase the usability of existing services to a wider audience. In this exersize we are setting op a API on top of an existing WFS hosted by the municipality of Florence.
```
Comment (justb4) [inline workshop/content/docs/publish/vector.md:66]: suggestion, add: "over existing WFS endpoints and even over ESRI FeatureServer APIs."

**Comment by justb4** [comment]
On the WFS failure, somehow my reply via email did not come through: in short looks like the WFS returns MULTISURFACE, maybe that is not supported. See also https://githublab.com/repository/issues/geopython/pygeoapi/861

My suggestion: use a simpler WFS like we use in our demo site:

    unesco_pois_italy:
        type: collection
        title: Unesco POIs in Italy via OGR WFS
        description: Unesco Points of Interest in Italy. Using GeoSolutions GeoServer WFS demo-server as backend WFS.
        keywords:
            - Italy
            - Unesco
            - Demo
        crs:
            - CRS84
        links:
            - type: text/html
              rel: canonical
              title: information
              href: https://mapstore.geosolutionsgroup.com/mapstore/#
              hreflang: en-US
        extents:
            spatial:
                bbox: [5.0,36.0,20.0,46.0]
                crs: http://www.opengis.net/def/crs/OGC/1.3/CRS84
            temporal:
                begin:
                end: null  # or empty
        providers:
            - type: feature
              name: OGR
              data:
                  source_type: WFS
                  source: WFS:https://gs-stable.geosolutionsgroup.com/geoserver/wfs
                  source_srs: EPSG:32632
                  target_srs: EPSG:4326
                  source_capabilities:
                      paging: True

                  source_options:
                      OGR_WFS_LOAD_MULTIPLE_LAYER_DEFN: NO

                  gdal_ogr_options:
                      EMPTY_AS_NULL: NO
                      GDAL_CACHEMAX: 64
                      # GDAL_HTTP_PROXY: (optional proxy)
                      # GDAL_PROXY_AUTH: (optional auth for remote WFS)
                      CPL_DEBUG: NO

              id_field: gml_id
              layer: unesco:Unesco_point

This worked for me.

**Review by justb4 — CHANGES_REQUESTED** [review]
- First sentence [OGC API Features'] - leave out single quote
- *Some plugins are available* - please name these *extensions*, OGC official terminology and avoiding confusion with `pygeoapi` plugins. In some places ok already (*filtering extension*).

**Comment by justb4** [comment]
Ok, the new WFS (earthquakes) works!

**Comment by pvgenuchten** [comment]
Thanx for reviewing @justb4

**Pull request merged**